libr/debug/plugin.c

```diff
@@ -50,27 +50,31 @@ R_API bool r_debug_use(RDebug *dbg, const char *str) {
 	return (dbg && dbg->h);
 }
 
-R_API int r_debug_plugin_list(RDebug *dbg, int mode) {
+R_API bool r_debug_plugin_list(RDebug *dbg, int mode) {
 	char spaces[16];
 	int count = 0;
 	memset (spaces, ' ', 15);
 	spaces[15] = 0;
 	RDebugPlugin *h;
 	RListIter *iter;
+	PJ *pj = NULL;
 	if (mode == 'j') {
-		dbg->cb_printf ("[");
+		pj = pj_new ();
```

Comment:
Better use core_pj_new() i thunk having a PJ instance in rdebug is the simplest solution here. So it can be initialized by core for user sefined options